Description: Migrate from usb_id and usb_db to udev builtins usb_id and hwdb.
Author: Dmitrijs Ledkovs <dmitrij.ledkov@ubuntu.com>
Bug-Ubuntu: https://bugs.launchpad.net/bugs/1200185

--- colord-1.0.1.orig/rules/69-cd-sensors.rules.in
+++ colord-1.0.1/rules/69-cd-sensors.rules.in
@@ -86,8 +86,8 @@ ATTRS{idVendor}=="273f", ATTRS{idProduct
 
 # color calibration device
 ENV{COLORD_SENSOR_KIND}=="*?", ENV{COLOR_MEASUREMENT_DEVICE}="1"
-ENV{COLORD_SENSOR_KIND}=="*?", ENV{ID_MODEL}=="", IMPORT{program}="usb_id --export %p"
-ENV{COLORD_SENSOR_KIND}=="*?", ENV{ID_MODEL_FROM_DATABASE}=="", IMPORT{program}="usb-db %p"
+ENV{COLORD_SENSOR_KIND}=="*?", ENV{ID_MODEL}=="", IMPORT{builtin}="usb_id"
+ENV{COLORD_SENSOR_KIND}=="*?", ENV{ID_MODEL_FROM_DATABASE}=="", IMPORT{builtin}="hwdb --subsystem=usb"
 
 # Allow the daemon to access the color devices
 ENV{COLORD_SENSOR_KIND}=="*?", GROUP="@daemon_user@"
